app/views/directives/logs/_log-viewer.html

```diff
@@ -53,16 +47,8 @@ <h2>Logs are not available.</h2>
     {{emptyStateMessage}}
   </p>
 
-  <div ng-if="kibanaAuthUrl">
-    <form
-      action="{{kibanaAuthUrl}}"
-      method="POST">
-      <input type="hidden" name="redirect" value="{{kibanaArchiveUrl}}">
-      <input type="hidden" name="access_token" value="{{access_token}}">
-      <button class="btn btn-primary btn-lg">
-        View Archive
-      </button>
-    </form>
+  <div ng-if="kibanaArchiveUrl">
+    <a href="{{kibanaArchiveUrl}}" class="btn btn-primary btn-lg">View Archive</a>
```

Comment:
`ng-href`

Why does this have button styling? It should just be a normal link.